docs/src/snap/howto/networking/alternative-cni.md

```diff
@@ -0,0 +1,104 @@
+# How to use an alternative CNI
+
+While {{product}} ships with a default CNI that we guarantee is compatible with
+our distribution, you may want to use a different CNI plugin for your specific
+networking requirements. This guide explains how to safely replace the default
+CNI with an alternative solution.
+
+## What you'll need
+
+This guide assumes the following:
+
+- You have root or sudo access to the machine.
+- Basic understanding of Kubernetes networking concepts.
+- Basic knowledge of Helm.
+
+## Disable default network implementation
+
+For an existing cluster, disable the default network
+plugin:
+
+```
+sudo k8s disable ingress gateway network
+```
+
+For a new cluster, create a bootstrap configuration that disables networking:
+
+```
+cat <<EOF > bootstrap-config.yaml
+cluster-config:
+  network:
+    enabled: false
+```
+
+Then bootstrap the cluster with this configuration:
+
+```
+sudo k8s bootstrap --file bootstrap-config.yaml
+```
+
+## Configure Helm repository
+
+Add the CNI's Helm repository to {{product}}'s Helm installation. This guide
+uses Calico as an example:
+
+```
+sudo k8s helm repo add projectcalico https://docs.tigera.io/calico/charts
+```
+
+## Install Calico CNI
```

Comment:
The title of the guide implies that this is a generic guide but with calico as an example. I would consider changing the title to reflect this - "Install alternative CNI"